pdi-engine-api/api/src/main/java/org/pentaho/di/engine/api/model/MaterializedModelElement.java

```diff
@@ -35,7 +35,7 @@
  */
 public interface MaterializedModelElement extends ModelElement {
 
-  <D extends Serializable> List<Publisher<? extends ReportingEvent>> getPublisher( Class<D> type );
+  <D extends Serializable> List<Publisher<? extends PDIEvent>> getPublisher( Class<D> type );
```

Comment:
@CodeOnCoffee what was the motivation for returning a List of Publishers?
If these publishers are only ever merged together, shouldn't the API return a single Publisher?